docker-compose.yml

```diff
@@ -49,6 +48,13 @@ services:
             - "4150"
         entrypoint: ["/nsqd"]
 
+    wiremock:
+        image: rodolpheche/wiremock
+        ports:
```

Comment:
i don't think you should expose the port, that'll remove the risk of port conflict when the test run on a machine that has that port allocated